nrf_dm/README.rst

```diff
@@ -3,15 +3,18 @@
 Nordic Distance Measurement library
 ###################################
 
-The Distance Measurement library measures the distance between nRF52 Series devices.
+The Distance Measurement library measures the distance between nRF52 Series and nRF53 Series devices.
 It does the following:
 
 * It provides an easy-to-use interface for measuring distances.
 * It computes the distance using all the information available to the device.
 * It computes the distance based on the measured differential RF physical channel frequency response (MCPD mode) or the real time flight of packets (RTT mode).
 * It performs all the mathematics necessary to compute the distance inside the library.
 
-The library is available as hard-float builds for the nRF52 Series, and it is included as a pre-built static library.
+The library is available as hard-float builds for the nRF52 Series.
+On the nRF53 Series, the library is available as soft-float build for the application core.
+For the network core of the nRF53 Series, there is a hard-float library which contains the computation heavy post-processing functionality only.
```

Comment:
```suggestion
For the network core of the nRF53 Series, there is a hard-float library that handles the computation-heavy post-processing functionality only.
```
I've also seen "compute-intensive" used. Would that be appropriate instead of "computation-heavy"?

Reply:
will change it 👍